frontend/eda/common/TeamAccess.tsx

```diff
@@ -0,0 +1,185 @@
+import { useTranslation } from 'react-i18next';
```

Comment:
Can we move this file under the access directory? Perhaps `frontend/eda/access/common`